### PR TITLE
Add HTTP semantic convention stability blog post

### PR DESCRIPTION
**Pull request by trask**
(for reference, see https://github.com/open-telemetry/opentelemetry-specification/issues/3112)

### CODE REVIEW

**Comment by cartermp** [comment]
Be still, my heart! 😍

File: content/en/blog/2023/http-conventions-stability/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+The OpenTelemetry community is beginning the final push to declare HTTP semantic
+conventions stable!
+
+Following the recently proposed
+[Semantic Convention Process](https://docs.google.com/document/d/1ghvajKaipiNZso3fDtyNxU7x1zx0_Eyd02OGpMGEpLE), 
+the HTTP semantic convention stability working group will be meeting three times
+a week for the next six weeks (starting January 30) to work through the
+remaining
+[list of issues](https://github.com/orgs/open-telemetry/projects/41/views/1).
+This group will be submitting spec PRs for any changes that need to be made
```
Comment (cartermp) [inline content/en/blog/2023/http-conventions-stability/index.md:17]:
```suggestion
This group will submit spec PRs for any changes that need to be made
```

File: content/en/blog/2023/http-conventions-stability/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+date: 2023-01-26
+author: "[Trask Stalnaker](https://github.com/trask) (Microsoft)"
+---
+
+The OpenTelemetry community is beginning the final push to declare HTTP semantic
+conventions stable!
+
+Following the recently proposed
+[Semantic Convention Process](https://docs.google.com/document/d/1ghvajKaipiNZso3fDtyNxU7x1zx0_Eyd02OGpMGEpLE), 
+the HTTP semantic convention stability working group will be meeting three times
```
Comment (cartermp) [inline content/en/blog/2023/http-conventions-stability/index.md:13]:
```suggestion
the HTTP semantic convention stability working group will meet three times
```

**Review by cartermp — APPROVED** [review]
Minor copy-edits. Very happy to announce this.

File: content/en/blog/2023/http-conventions-stability/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+And finally, the working group and spec approvers will have two weeks to clean
+up and merge the final spec PRs, and mark the HTTP semantic conventions as stable. 
+
+If you are interested in participating in the working group, please join the
+meetings starting Monday, January 30 at 3pm Pacific Time, and continuing for
+six weeks on Mondays, Wednesdays and Fridays: 
+
+* Mon 3-3:30
+* Wed 3-3:30
+* Fri 9:30-10
```
Comment (svrnm) [inline content/en/blog/2023/http-conventions-stability/index.md:32]: keep the info for each of them that it is in Pacific Time

File: content/en/blog/2023/http-conventions-stability/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: Final push to HTTP semantic convention stability
+linkTitle: HTTP semantic conventions
+date: 2023-01-26
+author: "[Trask Stalnaker](https://github.com/trask) (Microsoft)"
+---
+
+The OpenTelemetry community is beginning the final push to declare HTTP semantic
```
Comment (svrnm) [inline content/en/blog/2023/http-conventions-stability/index.md:8]: Link to the HTTP semantic conventions in the spec?

**Review by svrnm — CHANGES_REQUESTED** [review]
love it, some minor suggestions

**Review by chalin — APPROVED** [review]
✨

**Review by svrnm — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

**Comment by svrnm** [comment]
I will merge this one in before the PHP one, because it is time sensitive (the 30th is today!)

**Pull request merged**